modules/apps/change-tracking/change-tracking-spi/src/main/java/com/liferay/change/tracking/spi/display/BaseCTDisplayRenderer.java

```diff
@@ -115,6 +133,38 @@ protected void buildDisplay(DisplayBuilder<T> displayBuilder)
 		}
 	}
 
+	protected void buildTableContent(
```

Comment:
Make it a private method to avoid semantic version increase. This is also not meant to be overridden.